app/assets/javascripts/search.js

```diff
@@ -5,18 +5,21 @@ if($("#advanced-search").length){
   var $description = $('input#description');
   var $downloads   = $('input#downloads');
   var $updated     = $('input#updated');
+  var $yanked      = $('input#yanked');
 
-  $name.add($summary)
+    $name.add($summary)
     .add($description)
     .add($downloads)
     .add($updated)
+    .add($yanked)
     .on('input', function(e) {
       var name        = $name.val().length > 0 ? 'name: ' + $name.val() : '';
       var summary     = $summary.val().length > 0 ? 'summary: ' + $summary.val() : '';
       var description = $description.val().length > 0 ? 'description: ' + $description.val() : '';
       var downloads   = $downloads.val().length > 0 ? 'downloads: ' + $downloads.val() : '';
       var updated     = $updated.val().length > 0 ? 'updated: ' + $updated.val() : '';
+      var yanked      = $yanked.is(":checked") ? 'yanked: true' : '';
 
-      $main.val($.trim(name + ' ' + summary + ' ' + description + ' ' + downloads + ' ' + updated));
+        $main.val($.trim(name + ' ' + summary + ' ' + description + ' ' + downloads + ' ' + updated + ' ' + yanked));
```

Comment:
please don't change indentation, here and on line 10. make sure you editor is set for picking indentation automatically. we have used 2 spaces in all files.